vendor/redz/tlm-lw.yaml

```diff
@@ -0,0 +1,143 @@
+        # which is verbose.
+        vendorID: 
+        # Identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: tlm-lw-profile-868
+        lorawanCertified: true
+        codec: tlm-lw-codec
+      US902-928:
+        id: tlm-lw-profile-915
+        lorawanCertified: true
+        codec: tlm-lw-codec
+      AS923:
+        id: tlm-lw-profile-923
+        lorawanCertified: true
+        codec: tlm-lw-codec
+      IN865-867:
+        id: tlm-lw-profile-865
+        lorawanCertified: true
+        codec: tlm-lw-codec
+
+  -
+
+
+
+
+# Bridge interfaces (optional)
+# Valid values are: modbus, m-bus, can bus, rs-485, sdi-12, analog.
+bridgeInterfaces:
+  - modbus
+  - rs-485
+
+# Dimensions in mm (optional)
+# Use width, height, length and/or diameter
+dimensions:
+  width: 43
+  length: 124
+  height: 95
+
+# Weight in grams (optional)
+weight: 400
+
+
+
+# Operating conditions (optional)
+operatingConditions:
+  # Temperature (Celsius)
+  temperature:
+    min: -40
+    max: 85
+  # Relative humidity (fraction of 1)
+  relativeHumidity:
+    min: 0
+    max: 0.95
+
+# IP rating (optional)
+ipCode: IP52
+
+# Key provisioning (optional)
+# Valid values are: custom (user can configure keys), join server and manifest.
+keyProvisioning:
+  - custom
+  - join server
+
+# Key programming (optional)
+# Valid values are: bluetooth, nfc, wifi, serial (when the user has a serial interface to set the keys)
+# and firmware (when the user should change the firmware to set the keys).
+keyProgramming:
+  - serial
+  - firmware
+
+# Key security (optional)
+# Valid values are: none, read protected and secure element.
+keySecurity: none
+
+# Firmware programming (optional)
+# Valid values are: serial (when the user has a serial interface to update the firmware), fuota lorawan (when the device
+# supports LoRaWAN FUOTA via standard interfaces) and fuota other (other wireless update mechanism).
+firmwareProgramming:
+  - serial
+  - fuota other
+
+# Product and data sheet URLs (optional)
+productURL: https://redz-sc.com/en/product/tlm-series-lora-and-lorawan-modems/tlm354-868mhz-lorawan-endnode-modem-with-modbus-tcprtu-scheduler-with-2-x-10100base-tx-ports-1-x-rs232-and-1-x-rs485-serial-ports
+dataSheetURL: https://admin.redz-sc.com/uploads/brochure/tlm354-datasheet-en.pdf
+
+# Commercial information
+resellerURLs:
+  - name: 'Z Telemetri'
+    region:
+      - Worldwide
+    url: https://ztelemetry.com/en/
+  - name: 'Marcom'
+    region:
+      - Italy
+    url: https://shop.marcomweb.it/it/
+  - name: 'S Kommunikacja'
+    region:
+      - Poland
+    url: https://pomiary.pl/
+msrp:
+  USD: 280
+
+# Photos
+photos:
+  main: TLM-LW.png # Image needs to have a transparent background
+  other:
+    - TLM-LW-sc1.jpg # Image needs to have a transparent background
+    - TLM-LW-sc2.jpg # Image needs to have a transparent background
```

Comment:
You cannot use capital letters, change the name of the images to lowercase and change this.